### PR TITLE
[zh-cn]Update memory-default-namespace.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 1]
```diff
@@ -247,6 +247,21 @@ resources:
...
 
+<!--
+{{< note >}}
+
+A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be scheduleable.
+See [Constraints on resource limits and requests](/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests) for more details.
+
+{{< /note >}}
+-->
+{{< note >}}
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:258]: Please use this format instead.

```md
{{< note >}}
<!--
English content
-->
中文翻譯
{{< /note >}}
```

File: content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 2]
```diff
@@ -247,6 +247,21 @@ resources:
...
+
+`LimitRange` **不会**检查它应用的默认值的一致性。 这意味着 `LimitRange` 设置的 _limit_ 的默认值可能小于客户端提交给 API 服务器的声明中为容器指定的 _request_ 值。 如果发生这种情况，最终会导致 Pod 无法调度。
+更多信息，请参阅[资源限制的 limit 和 request](/zh-cn/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests)。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:261]: Please break line into 60 - 120 characters.
```suggestion
`LimitRange` **不会**检查它应用的默认值的一致性。 这意味着 `LimitRange` 设置的 _limit_ 的默认值可能小于客户端提交给
API 服务器的声明中为容器指定的 _request_ 值。如果发生这种情况，最终会导致 Pod 无法调度。更多信息，
请参阅[资源限制的 limit 和 request](/zh-cn/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests)。
```

**Review by Sea-n — CHANGES_REQUESTED** [review]

**Comment by Sea-n** [comment]
Hey @kinzhi,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by kinzhi** [comment]
> Hey @kinzhi,
> 
> It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

Yes, I have revised it

**Comment by tengqm** [comment]
@kinzhi Looks like the CI gate has some transient issues on this. Could you please recommit this PR to trigger another check?

**Comment by Sea-n** [comment]
Please solve merge conflict and `/unhold` by yourself.
/hold

**Pull request merged**